design/retry-patching-configuration_design.md

```diff
@@ -0,0 +1,109 @@
+# Backup Restore Status Patch Retrying Configuration
+
+## Abstract
+When a backup/restore completes, we want to ensure that the restore custom resource progress to the correct status.
```

Comment:
When a backup/restore completes, we want to ensure that the custom resource progresses to the correct status. ?